### PR TITLE
[DOCS] Added notice about required Python version

### PR DESCRIPTION
**Pull request by TheCedarPrince**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Users are welcomed, however, to utilize their own Synthea and/or OMOP vocabulary
 - See the top of [this page](https://docs.getdbt.com/docs/core/pip-install) for OS & Python requirements.  (Do NOT install dbt yet - see below for project installation and setup.)
 - It is recommended to use [VS Code](https://code.visualstudio.com/) as your IDE for developing this project.  Install the `dbt Power User` extension in VS Code to enjoy a plethora of useful features that make dbt development easier
 - This project currently only supports **Synthea v3.0.0**
+- Python **3.12.0** is the suggested version of Python for this project.
```
Comment (lawrenceadams) [inline README.md:26]: Great point! Is there any reason why you picked 3.12?

I think [dbt-duckdb](https://github.com/duckdb/dbt-duckdb/blob/master/setup.cfg) supports anything above version 3.8?

Comment (TheCedarPrince) [reply README.md:26]: Hey @lawrenceadams , yes as I grepped the repo trying to find the correct Python version to use and the only one I could find as a reference was stated here: https://github.com/OHDSI/dbt-synthea/blob/4c34141f301ad30601bd7c373bab3912da2c26f6/.github/workflows/generate-docs.yml#L19 although this is only for CI.

I didn't check package compat on other Python versions so not sure what other packages and package versions could work with different Python versions.

Comment (lawrenceadams) [reply README.md:26]: Ahhh makes sense -  I think I chose that one as it seemed like a logical starting point! Fair point

**Comment by TheCedarPrince** [comment]
Feel free to merge this -- seems good to go. :)

**Pull request merged**